roles/openshift_node/tasks/main.yml

```diff
@@ -133,11 +133,20 @@
   retries: 120
   delay: 1
   changed_when: false
-  when: openshift.common.is_containerized | bool
 
 - name: Start and enable node
   service: name={{ openshift.common.service_type }}-node enabled=yes state=started
   register: node_start_result
-
+  failed_when: false
+  
+- name: Sleep 10 seconds when starting the node failed
+  pause: secs=10
+  when: node_start_result.state != 'started'
+  
+- name: Try starting the node again
+  service: name={{ openshift.common.service_type }}-node enabled=yes state=started
+  register: node_start_result
+  when: node_start_result.state != 'started'
```

Comment:
We'd still the sleep call though, right?  It's hard to validate something that may disappear randomly in a 10 second window.  Whatever we decide we should debug this a little more and write up a good comment.  This definitely reads weird.